server/routes/bacalhau.js

```diff
@@ -1,38 +1,23 @@
-var express = require('express');
+var express = require("express");
 var router = express.Router();
-const { exec, spawn } = require('node:child_process');
+const { exec, spawn } = require("node:child_process");
 
-
-router.post('/', function(req, res, next) {
+router.post("/", function(req, res, next) {
   const body = req.body;
-  console.log(body.completedJobsArray)
-  const rawArray = body.completedJobsArray;
-  let newCommand = "";
-
-  for (var i = 0; i < rawArray.length; i++) {
-    console.log(rawArray[i]);
-    newCommand = newCommand + " " + rawArray[i] + " ";
-    console.log(newCommand);
-  }
+  console.log(body.completedJobsArray);
+  const cid = body.completedJobsArray;
```

Comment:
Can this be `body.cid` instead of `body.completedJobsArray`?

Reply:
yea